### DISCUSSION TITLE
How can I  prevent access to the Ant Media Server Web Panel?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5123] [FAQs] [answered]
How can I hide the Ant Media Server web panel from the public or anyone with a playback URL will be able to figure out how to get to the web panel login page?

Is there a way to restrict Web Panel access to specific IPs and still deliver playback URLs on 443 or 5443?

**Comment by Mohit-3196** [comment, marked as answer]
To restrict access to the Ant Media Server's web panel to specific IPs, you can follow these steps:

1. Open the server configuration file, which is located at `/usr/local/antmedia/conf/red5.properties`

2. Locate the property `server.allowed_dashboard_CIDR` within the configuration file. By default, it allows all IPs to access the web panel with the value `server.allowed_dashboard_CIDR=0.0.0.0/0`

3. Modify the configuration to specify the CIDR notation for the allowed IPs. For example, to allow access only from IP addresses 13.197.23.11/16 and 87.22.34.66/8, you would update the configuration as follows:
`server.allowed_dashboard_CIDR=13.197.23.11/16,87.22.34.66/8`
You can add multiple comma-separated CIDR notations if needed.

4. Save the changes to the configuration file.
5. Restart the Ant Media Server using the command: `sudo service antmedia restart` 

This will apply the new configuration, and only the specified IPs will be able to access the web panel.

Thank you
